### PR TITLE
Added functionality to create a convex hull for any trajectory, as an…

### PR DESCRIPTION
**Pull request by aleczoeller**
Added functionality to create a convex hull for any trajectory, as an object of the dataframe. Currently set to populate if generate(convex_hull=True) is used, or if set as df.convex_hull = df[['x','y']].  Added tests and documentation for calculating convex hull, determining whether a point is inside a polygon, and determining colinearity.

### CODE REVIEW

**Review by justinshenk — CHANGES_REQUESTED** [review]
Thanks for your PR! Can you please rebase your changes onto master. 
- Make sure you have the latest version of traja-team/traja master with `git pull upstream master` and then 
- Rebase: https://medium.com/@ruthmpardee/git-fork-workflow-using-rebase-587a144be470.

**Comment by aleczoeller** [comment]
> Thanks for your PR! Can you please rebase your changes onto master.
> 
>     * Make sure you have the latest version of traja-team/traja master with `git pull upstream master` and then
> 
>     * Rebase: https://medium.com/@ruthmpardee/git-fork-workflow-using-rebase-587a144be470.

This is rebased - I had built the documents (to html) in the repo when the only actual update was to the references file.  Please let me know if there's still an issue with extraneous changes.  Thanks!

File: traja/tests/test_trajectory.py [hunk 1]
```diff
@@ -221,17 +221,10 @@ def test_transition_matrix():
...
-    assert left_turns == 4
-    assert right_turns == 0
-
-
-def test_calc_flow_angles():
+def test_calculate_flow_angles():
     df_copy = df.copy()
     grid_indices = traja.grid_coordinates(df_copy)
-    U, V = traja.calc_flow_angles(grid_indices.values)
+    U, V = traja.calculate_flow_angles(grid_indices.values)
```
Comment (justinshenk) [inline traja/tests/test_trajectory.py:227]: @aleczoeller your fork does not show the most recent changes, like this shortening to “calc”. Please update your branch to reflect the latest master branch

Comment (aleczoeller) [reply traja/tests/test_trajectory.py]: Hi Justin, this has been updated.  It looked like there was a CI build failed test, but I couldn't see the specifics on the cause

File: traja/tests/test_trajectory.py [hunk 2]
```diff
@@ -221,13 +221,6 @@ def test_transition_matrix():
     transitions_matrix = traja.transition_matrix(grid_indices1D)
 
 
-def test_calc_laterality():
```
Comment (justinshenk) [inline traja/tests/test_trajectory.py:224]: It appears this test was unintentionally removed. Please restore it.

**Review by justinshenk — CHANGES_REQUESTED** [review]
One test method was removed 'test_calc_laterality'. Please restore it. Otherwise looks good!

File: traja/tests/test_trajectory.py (hunk 2 above)
Comment (aleczoeller) [reply traja/tests/test_trajectory.py]: Just re-added this test and pushed to this branch.  Thanks!

Comment (justinshenk) [reply traja/tests/test_trajectory.py]: Thanks!

**Pull request merged**